staging/src/k8s.io/kubectl/pkg/cmd/plugin/plugin.go

```diff
@@ -40,15 +40,19 @@ var (
 		Plugins provide extended functionality that is not part of the major command-line distribution.
 		Please refer to the documentation and examples for more information about how write your own plugins.
 
-		The easiest way to discover and install plugins is via the kubernetes sub-project krew.
-		To install krew, visit [krew.sigs.k8s.io](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)`))
+		The easiest way to discover and install plugins is via the kubernetes sub-project krew: [krew.sigs.k8s.io].
+		To install krew, visit https://krew.sigs.k8s.io/docs/user-guide/installing-plugins/`))
```

Comment:
I think, previous link is correct to install the Krew not installing the plugins.

Reply:
ah! right. I added link seeing the previous line stating how to install plugins using krew